imap_processing/cdf/cdf_attribute_manager.py

```diff
@@ -78,7 +78,9 @@ def __init__(self, data_dir: Path):
         self.global_attributes = CdfAttributeManager._load_yaml_data(
             self.source_dir / DEFAULT_GLOBAL_CDF_ATTRS_FILE
         )
-        self.variable_attributes = dict()
+        self.variable_attributes = dict()  # type: ignore[var-annotated]
+        # Todo Change, Need type annotation for "variable_attributes"
+        # (hint: "variable_attributes: dict[<type>, <type>] = ...")
```

Comment:
```suggestion
        self.variable_attributes: dict[str, dict] = dict()
```

I played around with this a bit. I think this is the solution.

Reply:
Worked on my end!